vector/src/main/java/im/vector/activity/VectorRoomCreationActivity.java

```diff
@@ -365,6 +361,87 @@ public static String isDirectChatRoomAlreadyExist(String aUserId, MXSession mSes
         return null;
     }
 
+
+    //==============================================================================================================
+    // Handle open existing direct chat room
+    //==============================================================================================================
```

Comment:
This section title is not formatted correctly

Reply:
I took the formatting already existing in the code. The word wrap problem comes from the size of the screen used to view the diff.

Reply:
I took the formatting already existing in the code. The word wrap problem comes from the size of the screen used to view the diff.